src/uvw/util.hpp

```diff
@@ -97,12 +103,12 @@ class Flags final {
 
     ~Flags() noexcept { static_assert(std::is_enum<E>::value, "!"); }
 
-    constexpr Flags& operator=(const Flags &f) noexcept {
+    R_CONSTEXPR Flags& operator=(const Flags &f) noexcept {
         flags = f.flags;
         return *this;
     }
 
-    constexpr Flags& operator=(Flags &&f) noexcept {
+    R_CONSTEXPR Flags& operator=(Flags &&f) noexcept {
```

Comment:
You can define this by means of the comma operator and work around the constraints on the `constexpr` for MSVC.  
It follows an example:
```
constexpr Flags& operator=(Flags &&f) noexcept {
    return flags = std::move(f.flags), *this;
}
```